app/build.gradle.kts

```
@@ -109,7 +109,7 @@ repositories {
 dependencies {
     val mockitoVersion = "3.12.4"
```

```suggestion
    val mockitoVersion = "5.14.2"
```

Update mockito to the newest version